src/components/AutoComplete/AutoComplete-story.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import Component from './AutoComplete';
+
+import sampleData from './data/sample';
+
+const AutoCompleteTest = () => {
+  const [result, setResult] = useState(null);
+
+  return (
+    <>
+      <label for={'autoCompleteField'}>AutoComplete list of countries</label>
+      <br />
+      <Component
+        id={'autoCompleteField'}
```

Comment:
No need for brackets for string literal
```suggestion
        id="autoCompleteField"
```